runtime/tools/python/ttrt/common/run.py

```diff
@@ -80,13 +84,15 @@ def initialize_api():
             default=False,
             choices=[True, False],
             help="do a golden identity test on the output tensors",
+            action="store_true",
         )
         Run.register_arg(
             name="--non-zero",
             type=bool,
             default=False,
             choices=[True, False],
             help="test the output tensors are non-zero",
+            action="store_true",
```

Comment:
what's the reason for adding action? were your new args not working without this. reason being is the other APIs also have bool options, so it we update it for run api, we should update it for all apis

Reply:
im indifferent to having it or not, but wondering if it wasn't working without it (since bool options should be captured correctly).

Reply:
I'll get rid of action, I added this specifically because a previous version of my change used the "append" action, but then we decided to use booleans, didn't remove the changes here

Reply:
awesome. everything else looks good. thank you!!